testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -17,6 +17,27 @@
 from MDAnalysis.topology.base import change_squash
 
 
+class TestGroupProperties(object):
+    """ Test attributes of all groups
+    """
+
+    def setUp(self):
+        self.u = make_Universe()
+        self.group_dict = {
+            'atom': self.u.atoms,
+            'residue': self.u.residues,
+            'segment': self.u.segments
+        }
+
+    def test_dimensions(self):
+        dimensions = np.arange(6)
+
+        for group in self.group_dict.itervalues():
+            group.dimensions = dimensions.copy()
+            assert_array_equal(group.dimensions, dimensions)
```

Comment:
should probably also do an `assert self.u.dimensions == dimensions` here to check that the change got propagated globally.